src/main/java/org/assertj/vavr/internal/Multimaps.java

```diff
@@ -0,0 +1,479 @@
+
+import static io.vavr.Predicates.not;
+import static org.assertj.core.error.ElementsShouldBe.elementsShouldBe;
+import static org.assertj.core.error.ShouldContain.shouldContain;
+import static org.assertj.core.error.ShouldContainAnyOf.shouldContainAnyOf;
+import static org.assertj.core.error.ShouldContainExactly.elementsDifferAtIndex;
+import static org.assertj.core.error.ShouldContainExactly.shouldContainExactly;
+import static org.assertj.core.error.ShouldContainKeys.shouldContainKeys;
+import static org.assertj.core.error.ShouldContainOnly.shouldContainOnly;
+import static org.assertj.core.error.ShouldContainOnlyKeys.shouldContainOnlyKeys;
+import static org.assertj.core.error.ShouldContainValue.shouldContainValue;
+import static org.assertj.core.error.ShouldContainValues.shouldContainValues;
+import static org.assertj.core.error.ShouldNotContain.shouldNotContain;
+import static org.assertj.core.error.ShouldNotContainKeys.shouldNotContainKeys;
+import static org.assertj.core.error.ShouldNotContainValue.shouldNotContainValue;
+import static org.assertj.core.internal.Arrays.assertIsArray;
+import static org.assertj.core.internal.CommonValidations.failIfEmptySinceActualIsNotEmpty;
+import static org.assertj.core.internal.CommonValidations.hasSameSizeAsCheck;
+import static org.assertj.core.util.Objects.areEqual;
+import static org.assertj.core.util.Preconditions.checkArgument;
+import static org.assertj.core.util.Preconditions.checkNotNull;
+import static org.assertj.vavr.api.ShouldNotContainValues.shouldNotContainValues;
+
+public final class Multimaps {
+
+    private static final Multimaps INSTANCE = new Multimaps();
+
+    private Failures failures = Failures.instance();
+
+    private Conditions conditions = Conditions.instance();
+
+    private Multimaps() {
+    }
+
+    public static Multimaps instance() {
+        return INSTANCE;
+    }
+
+    /**
+     * Verifies that the given {@code Multimap} contains the value for given {@code key} that satisfy given {@code valueCondition}.
+     *
+     * @param <K>            key type
+     * @param <V>            value type
+     * @param info           contains information about the assertion.
+     * @param actual         the given {@code Multimap}.
+     * @param key            the given key to check.
+     * @param valueCondition the given condition for check value.
+     * @throws NullPointerException if the given values is {@code null}.
+     * @throws AssertionError       if the actual map is {@code null}.
+     * @throws AssertionError       if the actual map not contains the given {@code key}.
+     * @throws AssertionError       if the actual map contains the given key, but value does not match the given {@code valueCondition}.
+     */
+    @SuppressWarnings("unchecked")
+    public <K, V> void assertHasEntrySatisfying(AssertionInfo info, Multimap<K, V> actual, K key,
+                                                Condition<? super V> valueCondition) {
```

Comment:
Why line wrapping?

Reply:
It was added by default by IDE and it is consistent with Maps class